javascript/main.js

```diff
@@ -0,0 +1,30 @@
+
+var button = document.getElementById("button");
+button.addEventListener("click", makeTree);
+document.addEventListener("keydown", enter);
+
+
+function enter(e) {
+  if (13 == e.keyCode) {
```

Comment:
While this works, this statement is considered a "yoda condition"

Try `if (e.keyCode == 13)`